portlets/src/main/frontend/src/apps/components/rule/ManageRules.vue

```diff
@@ -1,21 +1,22 @@
 <template>
     <section>
-     <b-alert v-if="addSuccess"
-                     :show="dismissCountDown"
-                     dismissible
-                     fade
-                     variant="success"
-                     @dismiss-count-down="countDownChanged">
-                <i class="uiIconSuccess"></i> Rule {{updateMessage}} successfully
-                </b-alert>
 
-        <b-alert v-if="addError"
-        class="alert alert-danger"
-          :show="dismissCountDown"
-          dismissible
-          fade
-          variant="danger"
-          @dismiss-count-down="countDownChanged"><i class="uiIconError"></i>  An error happen when adding a rule</b-alert>
+
+        <div class="alert alert-success" v-if="isadded || addSuccess" v-on:="closeAlert()">
+
+            <i class="uiIconSuccess"></i>
+            Rule {{updateMessage}} successfully
```

Comment:
not i18ned ?

Reply:
check : https://github.com/exo-addons/gamification/pull/20